huxley/www/static/js/huxley/components/CountrySelect.js

```diff
@@ -34,14 +33,22 @@ var CountrySelect = React.createClass({
 
   renderCommitteeOptions: function() {
     return this.props.countries.map(function(country) {
-      if (!country.special) {
+      var index = this.props.countryPreferences.indexOf(""+country.id)
+      if(index < 0){
         return (
           <option key={country.id} value={country.id}>
             {country.name}
           </option>
         );
+
+      }else{
+        return (
```

Comment:
Why are we returning an entire other element if it's disabled? Does the following not work?
```
return (
  <option key={country.id} value={country.id} disabled={index >= 0}>
    {country.name}
  </option>
);
```